test-cases/svelte-mode/a.original.svelte

```diff
@@ -0,0 +1,5 @@
+<script>
```

Comment:
These three lines are the same as `a.prettier.svelte`. How about this?
```
<script >
   let  name  =  "world"
 </script>
```

Reply:
Good idea, done.